tests/LANraragi/Plugin/Metadata/EHentai.t

```diff
@@ -0,0 +1,82 @@
+# LANraragi::Plugin::Metadata::EHentai
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
+
+use Cwd qw( abs_path );
+use Mojo::JSON qw(decode_json encode_json);
+use Mojo::File;
+
+use Test::More;
+use Test::Deep;
+
+my ($ROOT) = abs_path(__FILE__) =~ m/(.*)\/tests.+/;
```

Comment:
Not seeing the point of doing a regex on `__FILE__` here but I might be missing something 😛  
Wouldn't 
```
my $cwd = getcwd;
my $SAMPLES = "$cwd/tests/samples";
```
achieve a similar result?

Reply:
It's an habit of mine: like this it doesn't matter where I am when I call the script.

Reply:
Oh alright I see: I was asking since the tests for `Eze.pm` use `getcwd` and I'd rather keep things consistent. 😌